examples/pytorch/ggnn/README.md

```diff
@@ -0,0 +1,62 @@
+# Gated Graph Neural Network (GGNN)
+
+- Paper link: https://arxiv.org/pdf/1511.05493.pdf
+
+## Dependencies
+- PyTorch 1.0+
+
+- DGL 0.3.1+
+
+## Solving bAbI tasks
+
+- bAbI dataset is generated randomly by [script](https://github.com/facebook/bAbI-tasks). For convenience, we have generated the dataset we used in this example [here](https://s3.us-east-2.amazonaws.com/dgl.ai/models/ggnn_babi_data.zip). Unzip the file to current directory.
```

Comment:
Maybe you can add code to automatically handle this. See this [example](https://github.com/dmlc/dgl/blob/9c790b114326829044bb6fc8ac73869151210910/python/dgl/data/reddit.py#L12).